### PR TITLE
fix: issue 91 by changing 'delete' to 'deleteMany' and updating tests

### PR DESCRIPTION
**Pull request by geefuoco**
I updated the `destroy` function to use `deleteMany` instead of `delete`. Since it is using a unique id, it will only ever delete at most 1 session. 

I also updated the tests to reflect that change

### CODE REVIEW

**Comment by kleydon** [comment]
Hi @geefuoco - really appreciate the help!

Reviewing the PR now...

File: src/lib/prisma-session-store.ts [hunk 1]
```diff
@@ -214,7 +214,12 @@ export class PrismaSessionStore<M extends string = 'session'> extends Store {
       if (Array.isArray(sid)) {
         await Promise.all(sid.map(async (id) => this.destroy(id, callback)));
       } else {
-        await this.prisma[this.sessionModelName].delete({ where: { sid } });
+        // const session = await this.prisma[this.sessionModelName].findUnique({ where: { sid } });
```
Comment (kleydon) [inline src/lib/prisma-session-store.ts:217]: Would you mind removing the commented-out code,
and adding a comment to clarify that `deleteMany()` is being used to address a bug (with a link to the bug it is addressing) - just to make it clear for people reading this in the future why the "many" version is being used?

**Review by kleydon — CHANGES_REQUESTED** [review]
Looks like a great improvement, that will make a number of people relieved.

Just noticing that the project shouldn't really have two different lock files.
Since the project currently uses yarn, would you mind deleting package-lock.json?

**Comment by kleydon** [comment]
Looks like this is not yet passing the lint checks. (Running `yarn run lint` should reveal them...)

**Comment by kleydon** [comment]
@geefuoco - Found a couple of minor things I'm hoping you might be willing to address, but - clearly an improvement people will appreciate. Thanks again.

**Comment by geefuoco** [comment]
I will have a look and see if I can fix. Thanks

**Review by kleydon — APPROVED** [review]

**Review by kleydon — APPROVED** [review]

**Review by kleydon — APPROVED** [review]

**Pull request merged**